lib/util/mp4_box_parsers.js

```diff
@@ -251,6 +251,26 @@ shaka.util.Mp4BoxParsers = class {
     const codec = shaka.util.Mp4Parser.typeToString(fourcc);
     return {codec};
   }
+
+  /**
+   * Parses a HDLR box.
+   * @param {!shaka.util.DataViewReader} reader
+   * @return {!shaka.util.ParsedHDLRBox}
+   */
+  static parseHDLR(reader) {
+    reader.skip(8); // Skip "pre_defined"
+    const parseType = (data) => {
```

Comment:
This anonymous function doesn't make much sense to me.  It's only called once.  Why not just:

```js
const data = reader.readBytes(4);
let handlerType = '';
handlerType += String.fromCharCode(data[0]);
// ...
```

Alternately, we could add something like `readFourcc()` to DataViewReader.

Reply:
Changed!